package.xml

```diff
@@ -25,5 +25,6 @@
   <run_depend>talos_gazebo</run_depend>
   <run_depend>roscontrol_sot_talos</run_depend>
   <run_depend>dynamic_graph_bridge_msgs</run_depend>
-
+  <run_depend>sot-pattern-generator</run_depend>
+  <run_depend>talos_bauzil</run_depend>
```

Comment:
I can't find this package anywhere

Reply:
It is Thibaud's simulation of the bauzil room, but it is only on Gitlab for now because he has not exported it yet. I think the end goal was to integrate it in talos_data

Reply:
Ok, this project is protected (not public), then we can't make it a dependency of a public project yet. I'll see that with him.

Reply:
This project is now public, and released on robotpkg :)